src/index.ts

```diff
@@ -51,7 +51,7 @@ type RequiredStabilityOptions = {
 type StabilityOptions = RequiredStabilityOptions &
   Required<DraftStabilityOptions>
 
-type ImageData = { buffer: Buffer; filePath: string; seed: number; mimeType: string }
+type ImageData = { buffer: Buffer; filePath: string; seed: number; mimeType: string; requestId: string;}
```

Comment:
I don't think this really belongs in ImageData since it's common to all the results. Maybe in 'end' or a new 'start' event? But also if you care about the request ID I assume you could just generate it yourself and pass it in

Reply:
I was trying to deconstruct the filePath so I can move the files and rebuild a src for the img tag. I'm having a hard time getting the files to generate in a place I can access them :\ You can call me a noob!